recipe/recipe.yaml

```
@@ -18,14 +18,18 @@ build:
 
 requirements:
   host:
-    - python
+    - python ${{ python_min }}
     - pip
     - setuptools
   run:
-    - python
+    - python >=${{ python_min }}
     - numba >=0.59.1
 
 tests:
+  - requirements:
+     run:
+       - python ${{ python_min }}
+       - pip
```

Sorry I goofed on the indenting. Think this should fix it

```suggestion
  - requirements:
      run:
        - python ${{ python_min }}
        - pip
```